statsmodels/genmod/generalized_linear_model.py

```diff
@@ -791,7 +793,7 @@ def estimate_scale(self, mu):
                 return self._estimate_x2_scale(mu)
             elif self.scaletype.lower() == 'dev':
                 return (self.family.deviance(self.endog, mu, self.var_weights,
-                                             self.freq_weights, self.scale) /
+                                             self.freq_weights, scale) /
```

Comment:
Only based on spot checking
my guess is that scale here should be 1.
scale should not be an argument in estimate_scale